From 36776e4b764860cc473f3c95e197d01249f60f95 Mon Sep 17 00:00:00 2001
From: Huifeng Zhang <Huifeng.Zhang@arm.com>
Date: Tue, 7 Mar 2023 09:53:53 +0000
Subject: [PATCH] drivers: phy_mii: Remove the unneeded header file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove the 'soc.h' to fix the compiling error as the 'phy_mii.c'
doesn't need it.

Issue-id: SCM-5977
Signed-off-by: Huifeng Zhang <Huifeng.Zhang@arm.com>
Change-Id: I52867c57d253ae49c64bdf29578a8ee6735c46a2
Upstream-Status: Backport [main:3d58697d7d613a3f629f0e47113b727b5c6f1d16]
---
 drivers/ethernet/phy/phy_mii.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/ethernet/phy/phy_mii.c b/drivers/ethernet/phy/phy_mii.c
index 7441c3354fa..f05e761f795 100644
--- a/drivers/ethernet/phy/phy_mii.c
+++ b/drivers/ethernet/phy/phy_mii.c
@@ -11,7 +11,6 @@
 #include <zephyr/device.h>
 #include <zephyr/init.h>
 #include <zephyr/kernel.h>
-#include <soc.h>
 #include <zephyr/drivers/mdio.h>
 #include <zephyr/net/phy.h>
 #include <zephyr/net/mii.h>
-- 
2.25.1

